src/background/message-ports.js

```diff
@@ -44,19 +44,16 @@ export default function initializeMessagePorts() {
     case "add_item":
       return openDataStore().then(async (ds) => {
         const item = await ds.add(message.item);
-        broadcast({type: "added_item", item}, sender);
```

Comment:
Realized that since I've got the datastore now broadcasting these on the Logins API event handlers, all these messages got doubled up.